source/api/image/3.0/change-log.md

```diff
@@ -0,0 +1,150 @@
+
+JSON-LD remains the serialization format of the Image API, as it is for the Presentation API. Some features of the JSON-LD Community Group specification make significant improvements to the Presentation API's structure and consistency and are also adopted by the Image API although the impact is much less. While this specification is not a W3C Technical Recommendation at the time of release, the likelihood of the standardization process for JSON-LD 1.1 being successful is extremely high and the rewards have been judged to be worth the risk of unintended incompatibility. See issue [#1192](https://github.com/IIIF/api/issues/1192).
+
+
+## 2. Non-Breaking Changes
+
+The following changes are backwards compatible with version 2.1.1 of the Image API.
+
+### 2.1. Added notes on extension mechanisms and registry
+
+A new [Extensions][image30-extensions] section describes mechanisms for extension of image requests and the new [Extra Functionality][image30-extra-functionality] section describes how extensions are described in the image information response. The `extraQualities` and `extraFormats` properties have been added to allow description of additional functionality. See issues [#1374](https://github.com/IIIF/api/issues/1374), [#1373](https://github.com/IIIF/api/issues/1373), and [#1435](https://github.com/IIIF/api/issues/1435).
+
+There is now a [registry of known extensions][registry-extensions] to the IIIF specifications, which includes a [registry of Image API extensions][registry-image-extensions]. Extensions intended for community use should be registered in the extensions registry, but registration is not mandatory.
+
+### 2.2. Added `partOf` and `seeAlso` linking properties
+
+Two linking properties also present in the Presentation API were added to the image information. The `seeAlso` property addresses the use case of providing a link to technical metadata, and the `partOf` property supports discovery of a containing Manifest. See issue [#1507](https://github.com/IIIF/api/issues/1507) and [#600](https://github.com/IIIF/api/issues/600).
+
+### 2.3. Define JSON-LD profile for media type
+
+A specific media-type, to be used with the HTTP headers `Accept` and `Content-Type` was defined. This media type is the JSON-LD media type, with the context in the `profile` parameter, following established conventions. See issue [#1066](https://github.com/IIIF/api/issues/1066).
+
+
+## 3. Editorial Changes
+
+The following changes do not change the Image API behavior from version 2.1.1.
+
+#### 3.1. Clarify distinctions between underlying image content, full image, extracted region, and image returned
+
+The [Terminology][image30-terminology] section now explicitly defines **underlying image content** and **full image**. These terms, along with extracted region and image returned are used to describe the image manipulations more consistently than in previous versions. See issue [#1425](https://github.com/IIIF/api/issues/1425).
```

Comment:
should "extracted region" and "image returned" be in quotes or italicized to highlight that they are literal phrases that we use?

Reply:
👍 Was just looking at that, would vote for quotes

Reply:
OK, will add quotes